app/migrations/Version20180314145201.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20180314145201 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE adherents ADD emails_subscriptions LONGTEXT DEFAULT NULL COMMENT \'(DC2Type:simple_array)\'');
+        $this->addSql('UPDATE adherents SET emails_subscriptions = \'subscribed_emails_movement_information,subscribed_emails_government_information,subscribed_emails_weekly_letter,subscribed_emails_mooc,subscribed_emails_microlearning\' WHERE main_emails_subscription = TRUE');
```

Comment:
Keep in mind we have 500K lines in that table and these queries looks quite big.
Also a migration should not alter data IMO. Migrations are about schema changes.
Any existing data should be altered by a command or some SQL ran manually.
See with @atamano how he would like to handle this transition.

Last thing to know, if the migration failed in the middle, the database might be in an inconsistent state. See https://github.com/doctrine/DoctrineMigrationsBundle/issues/113